std_msgs/Header header

# joint names
string[] joint_name

# joint velocity readings
float64[] joint_vel
# Units defined by robot (rad, m, or normalized)

# Absolute joint position readings
float64[] joint_pos